input/pagecontent/reference.md

```diff
@@ -1010,20 +1010,26 @@ The referenced invariant and its properties must be declared somewhere within th
 
 #### Mapping Rules
 
-🚧 [Mappings](https://www.hl7.org/fhir/mappings.html) are an optional part of SDs that can be provided to help implementers understand the content and use resources correctly. These mappings are informative and are not to be confused with the computable mappings provided by [FHIR Mapping Language](https://www.hl7.org/fhir/mapping-language.html) and the [StructureMap resource](https://www.hl7.org/fhir/structuremap.html).
+[Mappings](https://www.hl7.org/fhir/mappings.html) are an optional part of SDs that can be provided to help implementers understand the content and use resources correctly. These mappings are informative and are not to be confused with the computable mappings provided by [FHIR Mapping Language](https://www.hl7.org/fhir/mapping-language.html) and the [StructureMap resource](https://www.hl7.org/fhir/structuremap.html).
 
 In FSH, mapping rules are not included in the profile definition.  Rather, they are included in a separate [Mapping definition](#defining-mappings) that provides additional context such as the higher level source and target. Within that definition mapping rules use the symbol `->` with the following grammar:
 
 ```
-* {path} -> {string}
+* -> "{map}" "{comment}" {mime-type}
+```
+
 ```
+* {path} -> "{map}" "{comment}" {mime-type}
```

Comment:
Should it be made clear that `{mime-type}` is a code?

Reply:
Added that comment "The mime type _is a code_ that must conform to https://tools.ietf.org/html/bcp13 (FHIR value set https://www.hl7.org/fhir/valueset-mimetypes.html)."